samigo/samigo-app/src/webapp/jsf/author/publishedSettings.jsp

```diff
@@ -182,10 +182,12 @@
           checkLastHandling();
         });
 		function expandAccordion(iframId){
-			$('.ui-accordion-content').show();
+			$("div#jqueryui-accordion > .ui-accordion-content").show();
 			mySetMainFrameHeight(iframId);
 			$("#collapseLink").show();
 			$("#expandLink").hide();
+			$("div#jqueryui-accordion > h3.ui-accordion-header > span").removeClass("ui-icon-triangle-1-e").addClass("ui-icon-triangle-1-s");
+			$("div#jqueryui-accordion > h3.ui-accordion-header").addClass("ui-accordion-header-active ui-state-active");
```

Comment:
Why don't you change the icon in collapseAccordion here, too? Is there no need?

Reply:
It doesn't appear to be necessary

Reply:
This is how we've been running it in our instance, and it's done the trick

Reply:
Ok, one more thing. I think you should move these functions into authoring.js. They are more or less identical. I know they weren't there in the first place, but it's a definite improvement in removing duplication. Never be afraid to improve code in the areas surrounding your bugfix.

Reply:
Not sure what you mean, are you saying get rid of them in publishedSettings and just leave them in authorSettings?

Reply:
No, I mean move the function definition into authoring.js so the functions are defined in one place. Both JSPs bring that file in, so you can do that and still call it from the two jsps.

Reply:
Oh right right, thanks. They are now moved to authoring.js

Reply:
Have you taken them out of the jsp files? It looks to me like they're still there. You can remove them from there and the call will use the definition from authoring.js.

Reply:
The jsp files are now amended with the functions in authoring.js